provider/balance_checker_test.go

```diff
@@ -146,5 +232,43 @@ func TestBalanceCheckerStartsWithdrawal(t *testing.T) {
 		t.Fatal("timed out waiting for completion")
 	}
 
-	testScaffold.queryClient.AssertExpectations(t)
+	testScaffold.qc.AssertExpectations(t)
+}
+
+func TestBalanceCheckerMonitorsFunds(t *testing.T) {
+	testScaffold, bc := leaseMonitorForTest(t, 1, time.Second*100)
+	defer testScaffold.testBus.Close()
+
+	subscriber, err := testScaffold.testBus.Subscribe()
+	require.NoError(t, err)
+	defer subscriber.Close()
+
+	lid := mtypes.LeaseID{
+		Owner:    bc.ownAddr.String(),
+		DSeq:     1,
+		GSeq:     0,
```

Comment:
I think these are normally >1 in the real network? 

I usually use a `testutil` function from our package, then overwrite the address fields.

Reply:
gseq/oseq are 0 by default. removing them